pkgs/tools/archivers/patool/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "patool";
+  version = "1.12";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "15qhrc2fl82pyp9w5s2s7d5cihm0zi988qpmysyfsfz1pzw0q673";
+  };
+
+  doCheck = false;
```

Comment:
this is a last resort, please check out source if the pypi doesn't have tests in it, and if neither have tests, then please give a comment saying so

Reply:
Okay, so `patool` *does* have tests, has some *really* complicated `requirements-test.txt`, that I was unable to reproduce. Should we skip it for now?

Reply:
Many of those error doesn't seem to be caused by missing package: I think that is rather because there should the tests/data folder, that doesn't (for any reason) doesn't exist. They should be removed when updating to pip, as they are present in the github repo.